pihole_adlist_tool

```diff
@@ -833,26 +837,27 @@ if [ "$FURTHER_ACTION" -eq 3 ]; then
     # remove all domains from gravity_dup that are also contained in that adlist
     # count how many domains are still on gravity_dup
 
-    adlist_conf_minimal_enabled=(`sqlite $TEMP_DB "select id from adlist where unique_domains_covered IS NOT NULL;"`)
+    mapfile -t adlist_conf_minimal_enabled < <(sqlite $TEMP_DB "select id from adlist where unique_domains_covered IS NOT NULL;")
 
     sqlite $TEMP_DB "CREATE TABLE gravity_dup AS SELECT * FROM gravity_strip"
 
     for adlist_id in "${adlist_conf_minimal_enabled[@]}"; do
        sqlite $TEMP_DB "DELETE FROM gravity_dup WHERE domain IN (SELECT domain from gravity_dup where adlist_id=$adlist_id);"
     done
 
-    LEFT_DOMAINS=(`sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;"`)
+    declare -i LEFT_DOMAINS
```

Comment:
It's declared in line 38

Reply:
Please resolve. As I sad `LEFT_DOMAINS` is already initialized in line 38. No need to declare it here.

Reply:
Done now in 70c76e399cb73ef9945d5e2f95b18f2b39044cd3